<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCefBrowserBar</class>
 <widget class="QWidget" name="QCefBrowserBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolButton_goBackward">
     <property name="toolTip">
      <string>Go back</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_goForward">
     <property name="toolTip">
      <string>Go forward</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_reload">
     <property name="toolTip">
      <string>Reload</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_stop">
     <property name="toolTip">
      <string>Stop load</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_url"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_go">
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
